http/handler.go

```diff
@@ -217,6 +218,7 @@ func newRouter(handler *Handler) *mux.Router {
 	router.HandleFunc("/index/{index}/field/{field}", handler.handlePostField).Methods("POST")
 	router.HandleFunc("/index/{index}/field/{field}", handler.handleDeleteField).Methods("DELETE")
 	router.HandleFunc("/index/{index}/field/{field}/import", handler.handlePostImport).Methods("POST")
+	router.HandleFunc("/index/{index}/field/{field}/importroaring/{shard}", handler.handlePostRoaringImport).Methods("POST")
```

Comment:
I think this should be `import-roaring` and the handler should be `handlePostImportRoaring`

Maybe elsewhere we drop the `Bytes` from `ImportRoaringBytes` for consistency.